src/eth.ts

```diff
@@ -0,0 +1,62 @@
+import ABI from './abi.json';
+import SNAPSHOT_ABI from './snapshotAbi.json';
+import { ethers } from 'ethers';
+import retry from 'async-retry';
+
+const MAX_CONCURRENCY = 200;
+const CONTRACT_ADDRESS = '0xf54cc94f1f2f5de012b6aa51f1e7ebdc43ef5afc';
+const SNAPSHOT_ADDRESS = '0xf3fB3F2Dab388Dc0d868be4A349aa1e8939D315D';
+
+export const createContract = function () {
```

Comment:
Add return types for all the functions here

Reply:
Updated! @NanopassKai